src/utils/staleness.js

```diff
@@ -34,16 +34,23 @@ class StalenessUtils {
 
   shouldUpdateStaleness(stalenessResponse, id) {
     const stalenessResponseTime = this.parseTime(stalenessResponse);
-    const { start } = this.openmct.time.getBounds();
-    const isStalenessInCurrentClock = stalenessResponseTime > start;
+    const { start, end } = this.openmct.time.getBounds();
```

Comment:
I don't think we ever need to consider the conductor bounds for staleness. So long as the new staleness message is newer than the last one, we should update staleness